build/codebase-check.sh

```diff
@@ -170,11 +170,16 @@ rc=0
 ARTIFACT_DIR=${ARTIFACT_DIR:-${PWD}}
 ERROR_FILE_NAME="codebase-errors.log"
 ERROR_FILE="${ARTIFACT_DIR}/${ERROR_FILE_NAME}"
+SUMMARY_FILE="${ARTIFACT_DIR}/summary-${ERROR_FILE_NAME}"
+OUTPUT_FILES="${ERROR_FILE} ${SUMMARY_FILE}"
```

Comment:
How did I not realize you could `tee` to multiple files? This is great!

Reply:
I know! I needed it to go to both files and I was like, "You know what would be great..." and `tee` had already solved my problem! 😄